forecasttools/idata_w_dates_to_df.py

```diff
@@ -0,0 +1,127 @@
+"""
+Contains functions for converting
+Arviz InferenceData objects to Polars
+dataframes with dates and draws and
+for working with conversions of
+Polars dataframes to Tidy dataframes.
+"""
+
+from datetime import datetime
+
+import arviz as az
+import numpy as np
+import polars as pl
+
+
+def add_dates_as_coords_to_idata(
```

Comment:
Just clarifying - this is intended to be a minimal version of this function, and we will add a more full-featured on soon?

Reply:
This will be addressed in #34